### DISCUSSION TITLE
Is version 1.8 officially out?

### DISCUSSION THREAD

**Discussion by DannyBen** [retypeapp/retype#113] [Q&A] [answered]
Installing retype gives version 1.8, and the [GitHub actions](https://github.com/retypeapp/action-build/blob/b000ba7cf31565de2ea0afa90446d91fddf4520a/build.sh#L3) seem to be updated with this version as well, but not the [releases](https://github.com/retypeapp/retype/releases).

Not sure if I can use it yet, across the whole pipeline.

**Comment by geoffreymcgill** [comment, marked as answer]
Yes, `v1.8.0` is released. I forgot to update the releases. We need to automate that process so it's not missed in the future. 

There will be a patch release coming soon as we might be adjusting how relative paths to folders are resolved. See #112. 

I'm running behind on the docs update too. More docs updates are coming today.

Sorry for the confusion. 

**UPDATE:** The [releases](https://github.com/retypeapp/retype/releases) have been updated.

**Reply by DannyBen** [reply]
Very cool! No apologies needed, I was just on the edge of my seat throughout the week, waiting to push my new `retype.yml` and say goodbye to its `json` ancestor, and didn't want to do so until the 1.8.0 ceremony is over.

**Reply by geoffreymcgill** [reply]
`v1.8.0` has been by far our largest release since `v1.0.0`. So much good stuff in there. I'm way behind with the documentation, but working my way through it step by step.

**Comment by noraj** [comment]
I'm waiting for a changelog 👀 

![image](https://user-images.githubusercontent.com/16578570/128486616-9bb0fa20-9afd-490b-9c42-24cad60b1610.png)

**Reply by geoffreymcgill** [reply]
The [roadmap](https://retype.com/roadmap) and [release](https://github.com/retypeapp/retype/releases) notes for `v1.8.0`, `v1.8.1` and `v1.8.2` have been improved.
